armi/reactor/tests/test_components.py

```diff
@@ -1075,6 +1076,62 @@ def test_dimensionThermallyExpands(self):
             self.assertEqual(cur, ref[i])
 
 
+class TestHexHoledCircle(TestShapedComponent):
+    componentCls = HexHoledCircle
+    componentDims = {
+        "Tinput": 25.0,
+        "Thot": 430.0,
+        "od": 16.5,
+        "holeOP": 3.6,
+        "mult": 1.0,
+    }
+
+    def test_getCircleInnerDiameter(self):
+        ref = 0  # there are multiple holes, so the function should return 0
+        cur = self.component.getCircleInnerDiameter(cold=True)
+        self.assertEqual(ref, cur)
+
+        # make and test another one with just 1 hole
+        simpleHexHoledCircle = HexHoledCircle(
+            "Void",
+            "circle",
+            self.componentDims["Tinput"],
+            self.componentDims["Thot"],
+            self.componentDims["od"],
+            self.componentDims["holeOP"],
+        )
+        self.assertEqual(
+            self.componentDims["holeOP"],
+            simpleHexHoledCircle.getCircleInnerDiameter(cold=True),
+        )
+
+    def test_getArea(self):
+        od = self.component.getDimension("od")
+        opHole = self.component.getDimension("holeOP")
```

Comment:
```suggestion
        holeOP= self.component.getDimension("holeOP")
```

Reply:
Ah yes, that's exactly what it is! Thank you, Mike!